lib/jsonnet/evaluator.rb

```diff
@@ -0,0 +1,46 @@
+module Jsonnet
+  class Evaluator
```

Comment:
How about something like this?
https://github.com/yugui/ruby-jsonnet/commit/4eca92cd7b87c2f1a1d6749266d8015f5d730448